freud/plot.py

```diff
@@ -7,7 +7,7 @@
 import warnings
 
 try:
-    from matplotlib.figure import Figure
+    import matplotlib.pyplot as plt
```

Comment:
@vyasr FYI, I fixed this issue we discussed offline a while back. Using `pyplot` ensures that the figure is "managed" and can be displayed.